.travis.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       php: 7.3
     - env: DB=sqlite; MW=REL1_34; SMW=~3.1@dev; PHPUNIT=6.5.*
       php: 7.2
-    - env: DB=mysql, MW=REL1_34; SMW=*3.2@dev; PHPUNIT=6.5.*
+    - env: DB=mysql, MW=REL1_34; SMW=~3.2@dev; PHPUNIT=6.5.*
```

Comment:
Can you spot the issue?

Hint: (https://travis-ci.org/github/SemanticMediaWiki/SemanticMetaTags/jobs/685542356)


`[60d904bc24ba516e5e7fff1d] [no req]   Error from line 582 of /home/travis/build/SemanticMediaWiki/mw/includes/installer/Installer.php: Class 'Mysql,Installer' not found`

Reply:
I saw this but at the time and even now I do not really know what to do with it. Looking at it again I suspect that code changes in SMW 3.2 made this extension somehow incompatible. This however does not explain why they run nicely together.

Reply:
Instead of `DB=mysql,` it needs to be `DB=mysql;` (the comma is the issue).

Reply:
This was too easy. I the heat of the action. :(

Reply:
If I was a programmer `Class 'Mysql,Installer' not found` is probably a good hint. For me the class name and related schemes do not mean anything. :|